Directory.Build.props

```diff
@@ -19,4 +19,9 @@
   <ItemGroup>
     <EmbeddedResource Include="**\*.rd.xml" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Pomelo.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
```

Comment:
FWIW, for MySqlConnector I had to add `<PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>` for CI running on non-Windows machines. However, this may have been a workaround for a non-Windows build issue that has already been fixed in .NET Core. Just bringing this up in case you get build errors and this might be a fix.

Reply:
Thanks for mentioning that. This seemed to be an issue in the past, but appears to have been fixed in the meantime. See https://github.com/roji/Npgsql/commit/3deea18b8a1836a8eef767e8ef899886eacb9aac